doc/sphinx/language/core/basic.rst

```diff
@@ -523,24 +523,24 @@ they appear after a boldface label.  They are listed in the
 Locality attributes supported by :cmd:`Set` and :cmd:`Unset`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The :cmd:`Set` and :cmd:`Unset` commands support the :attr:`local`,
-:attr:`global` and :attr:`export` locality attributes:
-
-* no attribute: the original setting is *not* restored at the end of
-  the current module or section.
-* :attr:`local` (or alternatively, the ``Local`` prefix): the setting
+The :cmd:`Set` and :cmd:`Unset` commands support the mutually
+exclusive :attr:`local`, :attr:`export` and :attr:`global` locality
+attributes:
```

Comment:
Do we need the full description of effects here? We could just reference to `_controlling-locality-of-commands` with a note that Set also has special alias syntax `Export`

Reply:
Indeed, now that we have a proper documentation of all locality attributes, we do not need to repeat it here (except for the weird default locality). Adding an explicit attribute for the default locality (as proposed in #13832) would even solve this remaining issue.

Reply:
I've applied your suggestion. Now this part of the text doesn't repeat the semantics of `local`, `export` and `global`.